packages/bootstrap/scss/timeselector/_variables.scss

```diff
@@ -8,10 +8,12 @@ $time-selector-bg: $component-bg !default;
 $time-selector-text: $component-text !default;
 $time-selector-border: $component-border !default;
 
-$time-selector-header-padding-x: k-map-get( $spacing, 1 ) !default;
-$time-selector-header-padding-y: k-map-get( $spacing, 1 ) !default;
+$time-selector-header-padding-x: map-get( $spacing, 4 ) !default;
```

Comment:
from this PR on we should use k-map-get instead of map-get: https://github.com/telerik/kendo-themes/pull/4030/files